tests/optim/param/test_transforms.py

```diff
@@ -305,6 +391,27 @@ def test_to_rgb_alpha_i1i2i3_forward(self) -> None:
 
         assertArraysAlmostEqual(inverse_tensor.numpy(), inverse_array)
 
+    def test_to_rgb_custom_forward(self) -> None:
+        if torch.__version__ == "1.2.0":
+            raise unittest.SkipTest(
+                "Skipping ToRGB forward due to insufficient Torch version."
+            )
+        matrix = torch.eye(3, 3)
+        to_rgb = transform.ToRGB(transform_matrix=matrix)
+        test_tensor = torch.ones(3, 4, 4).unsqueeze(0).refine_names("B", "C", "H", "W")
+        rgb_tensor = to_rgb(test_tensor)
+
+        to_rgb_np = numpy_transforms.ToRGB(transform_matrix=matrix.numpy())
+        test_array = np.ones((1, 3, 4, 4))
+        rgb_array = to_rgb_np.to_rgb(test_array)
+
+        assertArraysAlmostEqual(rgb_tensor.numpy(), rgb_array)
+
+        inverse_tensor = to_rgb(rgb_tensor.clone(), inverse=True)
+        inverse_array = to_rgb_np.to_rgb(rgb_array, inverse=True)
+
+        assertArraysAlmostEqual(inverse_tensor.numpy(), inverse_array)

```

Comment:
For this one as well - It would be better to assert with and expected hardcoded value just in case we have a bug in both implementation. Originally I thought that numpy has built-in tested functionality that we can assert with but since we are implementing both from scratch we can have bugs in both.

Reply:
Okay, I've replace the NumPy asserts in the forward tests with the exact expected tensor values.